UPGRADE-2.7.md

```diff
@@ -477,3 +477,10 @@ PropertyAccess
 
         new UnexpectedTypeException($value, $path, $pathIndex);
    ```
+
+Config
+------
+
+ * The `__toString()` method of the `\Symfony\Component\Config\ConfigCache` is marked as 
```

Comment:
deprecations don't belong in the UPGRADE file, instead it should be added in the CHANGELOG file of the Config component and the UPGRADE-3.0.md file should document that this method was removed from the interface.